### DISCUSSION TITLE
hires img2img with sd1.5 ?

### DISCUSSION THREAD

**Discussion by AlexD81** [blepping/comfyui_jankdiffusehigh#4] [Q&A] [answered]
Hi, I'm looking for a tool for sd1.5   to work with resolutions like 3200   with img2img. Where input img already has 3200 pixels resolution.     Will this addon help, or its purpose is different? Hard to understand from description.   

 when I try to run it

![изображение](https://github.com/user-attachments/assets/a08450f1-51c4-4fe5-bc60-3d1463bba3e0)

I'm getting following error  
![изображение](https://github.com/user-attachments/assets/384f4eac-2f1a-45db-9670-3dbd11b85acb)

**Comment by blepping** [comment]
> Will this addon help, or its purpose is different?

"work with" is pretty general but it definitely can be used for img2img workflows. if you turn off guidance (by setting either guidance steps or guidance factor to 0) then it's basically just normal highres fix.

> Where input img already has 3200 pixels resolution.

note that it will upscale the image by `scale_factor` at the start of each iteration so if you're passing it a pre-upscaled image than you just want to run img2img steps on then you're going to want to set `scale_factor` to `1.0` - otherwise it'll upscale your 3200x3200 image to 4800x4800 before running any steps. 

> I'm getting following error

unfortunately that's not enough information to diagnose the problem. please include the full text of the traceback from your console log or a workflow that can reproduce your issue.

**Comment by AlexD81** [comment]
Thanks for the tips .  
Tried again, after updating comfy, dont see errors anymore.

**Reply by blepping** [reply, marked as answer]
good to hear, glad you got it working! if you have any more questions or run into problems, please feel free to create a new issue or discussion. i've finished the main features i had planned so there should (hopefully) be less workflow breaking changes going forward.
